lib/quicly.c

```diff
@@ -2492,6 +2492,16 @@ static ssize_t round_send_window(ssize_t window)
     return window;
 }
 
+static inline uint64_t calc_remain_3x(quicly_conn_t *conn)
```

Comment:
How about somethign like `amplification_limit_allowance()` instead? Just because we might want to change the 3x multiplier.

Reply:
Done in f8e46c2.